.ci.yaml

```diff
@@ -130,7 +130,7 @@ platform_properties:
           {"dependency": "gems"},
           {"dependency": "ios_signing"}
         ]
-      os: Mac-10.15
+      os: Mac-12.0
```

Comment:
Should we use ""Mac-12.0.1" https://cs.opensource.google/flutter/infra/+/main:config/devicelab_staging_config.star;l=19?

Reply:
The bot dimension allows us to use either. From a config standpoint, it's easier to ignore hot fix versions and just use major.minor to gate tests